pkg/wgembed/management.go

```diff
@@ -38,14 +39,23 @@ func (wg *commonInterface) AddPeer(publicKey string, presharedKey string, addres
 		parsedAddresses = append(parsedAddresses, *allowedIPs)
 	}
 
+	udpEndpoint, _ := net.ResolveUDPAddr("udp", endpoint)
```

Comment:
Any reason to ignore the `error` here and do some custom error handling instead?

Reply:
there wasnt an error (i think?), for example if we entered `test.com:1234` it would return the udpEndpoint `:0`,

But will give it another go and maybe there was an error and i just didnt check it